### ISSUE TITLE
Cant Deactivate work order option in Odoo 12 Community 

### ISSUE THREAD

**Issue by bhavik1989863** [odoo/odoo#46238]
Impacted versions: Odoo 12 community

Steps to reproduce: Enable Work Order option from Manufacturing -> Settings. And than deactivate

Current behavior:
Even after you deactivate the Work order option . Manufacturing Order still expect to create users work order and process through that.

Expected behavior:

Work order option should be removed from the manufacturing order after we disable the option.

Video/Screenshot link (optional):
I check the code . And found the following note in the code.
      
"If we activate 'MRP Work Orders', it means that we need to install 'mrp_workorder'.
The opposite is not always true: other modules (such as 'quality_mrp_workorder') may
depend on 'mrp_workorder', so we should not automatically uninstall the module if 'MRP
Work Orders' is deactivated.
Long story short: if 'mrp_workorder' is already installed, we don't uninstall it based on
group_mrp_routings"

Seems logical. But in my case i activate that option just give idea to my client. And now client want Manufacturing order to be simple , without work order , so i am not sure how to remove work order option.

**Comment by bhavik1989863** [comment]
Closing since i realized that, it is dependant on the route, if mo have route , than work order is needed otherwise not.
